src/14_cal.py

```diff
@@ -29,4 +29,35 @@
 
 import sys
 import calendar
-from datetime import datetime
\ No newline at end of file
+from datetime import datetime
```

Comment:
Sys.argv should be used instead of terminal input, here is a link https://www.pythonforbeginners.com/system/python-sys-argv